packages/react/etc/react-compat.api.md

```diff
@@ -4,8 +4,294 @@
 
 ```ts
 
+import { IBaseProps } from '@fluentui/utilities';
+import { IComponentAs } from '@fluentui/utilities';
+import { IFocusZoneProps } from '@fluentui/react-focus';
+import { IRectangle } from '@fluentui/utilities';
+import { IRefObject } from '@fluentui/utilities';
+import { IRenderFunction } from '@fluentui/utilities';
+import { IStyle } from '@fluentui/style-utilities';
+import { IStyleFunctionOrObject } from '@fluentui/utilities';
+import { ITheme } from '@fluentui/style-utilities';
+import { KeyCodes } from '@fluentui/utilities';
+import { Point } from '@fluentui/utilities';
+import * as React from 'react';
+import { Target } from '@fluentui/react-hooks';
+
+// @public (undocumented)
+export class ActionButton extends React.Component<IButtonProps, {}> {
+    // (undocumented)
+    render(): JSX.Element;
+}
+
+// @public (undocumented)
+export class BaseButton extends React.Component<IBaseButtonProps, IBaseButtonState> implements IButton {
```

Comment:
now we have removed from `react-button` export from `@fluentui/react`, what's the reason to keep the compat layer?

Reply:
Not sure, aside from that it's possible people could have already started importing it. The benefit of removing would be clarity, plus that this is another area where we may ultimately go with a different solution. @khmakoto what do you think?

Reply:
Talked to Mak and he agrees that we should remove the compat layer (but not in this PR).

Reply:
I guess the other thing where we might need the compat layer in v8 is if people were using the old DatePicker and Calendar and are unable to migrate to the new ones right away. However this shouldn't be quite as bad of an issue since it's possible to pick up the change within v7 (by switching to importing from `@uifabric/date-time`).